documentation/objects/objects.md

```diff
@@ -17,49 +17,46 @@ weight: 1
 
 As the name suggests, the object classification workflow aims to classify full *objects*, based on object-level features and user annotations.
 An *object* in this context is a set of pixels that belong to the same instance.
-In order to do so, the workflow needs *segmentation* images besides the usual raw image data, that can e.g. be generated with the [Pixel Classification Workflow].
-Depending on the availability of these segmentation images, the user can choose between three flavors of object classification workflow, which differ by their input data:
+Object classification requires a second input besides the usual raw image data: an image that classifies the pixels as either belonging to an object or not, i.e. object predictions or labels.
```

Comment:
```suggestion
Object classification requires a second input besides the usual raw image data: an image that indicates for pixels whether they belong to an object or not, i.e. object predictions or a label image.
```
I would try to call _label image_ always _label image_, so that we can contrast this with _object labels_ (so whatever you label as a user)...